routes/web.php

```diff
@@ -33,13 +26,9 @@
     return view('matchdays.edit');
 });
 
-Route::get('/termine/create', function () {
-    return view('matchdays.create');
-});
+Route::get('/termine/create', 'MatchdayController@create');
```

Comment:
Wouldn't it be better to translate all parts of your url into english? e.g. /dates/create